src/pages/_app.tsx

```diff
@@ -1,14 +1,19 @@
+import 'core/polyfills';
+
```

Comment:
What is this used for here?

Reply:
This was a part I forgot about when switching to Next. This imports and defines some globals, for example to make `fetch` work in NodeJS, since that is part of the browser/dom-API and doesn't actually exist by itself in NodeJS